content/id/docs/contribute/_index.md

```diff
@@ -0,0 +1,79 @@
+---
+content_template: templates/concept
+title: Berkontribusi ke Dokumentasi Kubernetes
+linktitle: Berkontribusi
+main_menu: true
+weight: 80
+---
+
+{{% capture overview %}}
+
+Jika kamu ingin membantu dengan berkontribusi ke dokumentasi atau situs web Kubernetes, kami
+dengan senang hati menerima bantuan kamu! Siapapun bisa berkontribusi, baik kamu yang masih
+baru atau sudah lama di proyek ini, maupun jika kamu adalah seorang developer, seorang pengguna,
+atau bahkan seorang yang tidak tahan melihat saltik (_typo_)!
+
+Untuk informasi mengenai isi dan gaya (penulisan)
+dokumentasi Kubernetes, lihat [ikhtisar gaya penulisan dokumentasi](/docs/contribute/style/).
+
+{{% capture body %}}
+
+## Jenis-jenis kontributor dokumentasi
+
+- Seorang _member_ dari organisasi Kubernetes yang telah [menandatangani CLA](/docs/contribute/start#sign-the-cla)
+  dan berkontribusi waktu dan usahanya untuk proyek ini. Lihat
+  [Keanggotaan komunitas](https://github.com/kubernetes/community/blob/master/community-membership.md)
+  untuk kriteria spesifik untuk keanggotaan.
+- Seorang SIG Docs _reviewer_ adalah seorang anggota organisasi Kubernetes yang telah
+  menunjukkan ketertarikannya untuk memeriksa _pull request_ dokumentasi dan telah
+  ditambahkan ke dalam grup GitHub yang sesuai dan berkas-berkas `OWNERS` di dalam
+  repositori GitHub, oleh seorang SIG Docs Approver.
+- Seorang SIG Docs _approver_ adalah seorang anggota yang memiliki predikat
+  yang baik dan telah menunjukkan komitmen berkelanjutan terhadap proyek ini.
+  Seorang _approver_ dapat melakukan _merge_ terhadap _pull request_ dan
+  mempublikasi konten atas nama organisasi Kubernetes.
+  Para _approver_ juga dapat mewakili SIG Docs dalam komunitas Kubernetes
+  yang lebih besar. Beberapa tugas seorang SIG Docs _approver_, seperti
+  mengkoordinasi sebuah perilisan, membutuhkan komitmen waktu yang signifikan.
+
+## Cara-cara untuk berkontribusi ke dokumentasi
+
+Daftar ini dibagi menjadi hal-hal yang dapat dilakukan oleh siapapun, hal-hal yang
+dapat dilakukan oleh anggota-anggota organisasi Kubernetes, dan hal-hal yang
+memerlukan tingat akses yang lebih tinggi serta pengetahuan terhadap proses-proses
```

Comment:
typo
```suggestion
memerlukan tingkat akses yang lebih tinggi serta pengetahuan terhadap proses-proses
```

Reply:
Fixed in https://github.com/kubernetes/website/pull/18452/commits/7ca1ba41e206ce53601506378b5c886d22ed3905

Thanks!